Makefile

```
@@ -1,6 +1,8 @@
 SHELL := /bin/bash
 
-.PHONY: default clean build fmt lint shellcheck abcgo openapi-check style run test cover integration_tests license before_commit help godoc install_docgo install_addlicense install_golangci-lint
+.PHONY: default clean build golangci-lintfmt lint shellcheck abcgo openapi-check style run test \
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)